grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx

```diff
@@ -55,6 +58,8 @@ function IncidentStatusTag({
   );
 }
 
+const CUSTOM_VALUE_NUM = -100;
```

Comment:
Just to differentiate between all other values (-1 is taken for forever, all other positive values are matching values)